src/components/Facility/DoctorVideoSlideover.tsx

```diff
@@ -295,27 +294,10 @@ function UserListItem({ user }: { user: UserAnnotatedWithGroup }) {
           )}
           <div className="flex justify-between gap-2 text-sm text-secondary-500">
             <div className="flex items-center gap-1">
-              <a
-                role="button"
-                href="#"
-                onClick={(e) => {
-                  e.stopPropagation();
-                  copyToClipboard(user?.alt_phone_number || "");
-                  setCopied(true);
-                  setTimeout(() => setCopied(false), 2500);
-                }}
-              >
-                <span className="tooltip" id="copy-phoneicon">
-                  <span className="tooltip-text tooltip-top">
-                    {t("copy_phone_number")}
-                  </span>
-                  <CareIcon
-                    icon={copied ? "l-check" : "l-clipboard"}
-                    id="copy-icon"
-                    className="h-5 w-5"
-                  />
-                </span>
-              </a>
+              <CopyButton
+                content={user.alt_phone_number || ""}
```

Comment:
can we disable the button if there's no content?

```suggestion
                content={user.alt_phone_number}
```